<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemControl</class>
 <widget class="QWidget" name="SystemControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#404E5C;
}</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}</string>
   </property>
   <property name="text">
    <string>Face Detection Models</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}</string>
   </property>
   <property name="text">
    <string>Head Pose Model</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}</string>
   </property>
   <property name="text">
    <string>Eye Gaze Model</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="trialLineedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border: 2px solid #4F6272;
border-radius:20px;
color:white;
padding-left:10px;
	background-color: rgb(79, 98, 114);
}
QLineEdit:Hover{
border: 2px solid #BFD7EA;
}
QLineEdit:focus{
border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendtrialbutton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>630</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4F6272; 
color:white;
border-radius:20px;
}

QPushButton:Hover{
background-color:#BFD7EA
}</string>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}</string>
   </property>
   <property name="text">
    <string>System ON/OFF</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>108</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:white;
}</string>
   </property>
   <property name="text">
    <string>Model Selection</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="offbutton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4F6272; 
color:white;
border-radius:20px;
}

QPushButton:Hover{
background-color:#BFD7EA
}</string>
   </property>
   <property name="text">
    <string>OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onbutton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>80</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:#4F6272; 
color:white;
border-radius:20px;
}

QPushButton:Hover{
background-color:#BFD7EA
}</string>
   </property>
   <property name="text">
    <string>ON</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboFaceDetection">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 2px solid #4F6272;
border-radius:20px;
color:white;
padding-left:10px;
	background-color: rgb(79, 98, 114);
}
QComboBox:Hover{
border: 2px solid #BFD7EA;
}
QComboBox:focus{
border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboHeadPose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 2px solid #4F6272;
border-radius:20px;
color:white;
padding-left:10px;
	background-color: rgb(79, 98, 114);
}
QComboBox:Hover{
border: 2px solid #BFD7EA;
}
QComboBox:focus{
border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboEyeGaze">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
border: 2px solid #4F6272;
border-radius:20px;
color:white;
padding-left:10px;
	background-color: rgb(79, 98, 114);
}
QComboBox:Hover{
border: 2px solid #BFD7EA;
}
QComboBox:focus{
border: 2px solid white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="border_labe">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>671</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid #4F6272;
border-radius:20px;
padding-left:10px;
}
QLabel:Hover{
border: 2px solid #BFD7EA;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="border_labe_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>231</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
border: 2px solid #4F6272;
border-radius:20px;
padding-left:10px;
}
QLabel:Hover{
border: 2px solid #BFD7EA;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>border_labe</zorder>
  <zorder>border_labe_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>trialLineedit</zorder>
  <zorder>sendtrialbutton</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>offbutton</zorder>
  <zorder>onbutton</zorder>
  <zorder>comboFaceDetection</zorder>
  <zorder>comboHeadPose</zorder>
  <zorder>comboEyeGaze</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
